db/migrate/20151017171417_rpush_2_0_0_updates.rb

```diff
@@ -0,0 +1,67 @@
+class Rpush200Updates < ActiveRecord::Migration
+  module Rpush
+    class App < ActiveRecord::Base
+      self.table_name = 'rpush_apps'
+    end
+
+    class Notification < ActiveRecord::Base
+      self.table_name = 'rpush_notifications'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.